awscurl/awscurl.py

```diff
@@ -210,23 +296,22 @@ def sha256_hash_for_binary_data(val):
     # signed_headers, as noted earlier. Order here is not significant.
     # Python note: The 'host' header is added automatically by the Python
     # 'requests' library.
-    headers.update({
+    headers = {
         'Authorization': authorization_header,
```

Comment:
Would that drop other headers?

Reply:
Ah, I see it was moved, then function name is confusing , please fix

Reply:
Ah thanks, good catch.  Fixed function names and variable names to more accurately describe what's going on.  Also don't take `headers` as an argument any more because that was also confusing.